components/utilities/helpful.js

```diff
@@ -213,12 +213,17 @@ const Helpful = ({ slug, sourcefile }) => {
         method="POST"
         data-netlify="true"
         ref={formRef}
-        data-netlify-honeypot="bot-field"
+        data-netlify-honeypot="definitely-required-field"
```

Comment:
I gave this a fun name that doesn't mention "bot", just in case the bot is smart enough to ignore fields that say "bot".

I still worry about the mention of the term "honeypot", but there's nothing I can do about that.